kedro/framework/cli/project.py

```diff
@@ -412,6 +438,16 @@ def run(
     tag = _get_values_as_tuple(tag) if tag else tag
     node_names = _get_values_as_tuple(node_names) if node_names else node_names
 
+    # temporary duplicates for the plural flags
+    tags = _get_values_as_tuple(tags) if tags else tuple(tags)
```

Comment:
Minor, but do we need these `if/else` clauses? Since `_get_values_as_tuple("") == tuple("") == tuple()` already?

Reply:
Good catch - was following convention from the earlier implementation but looks like they weren't needed there either. My suspicion is that `multiple=True` implies a default value of an empty tuple 🤔

Reply:
Cool, that's what I suspected about `multiple=True` 👍  Just to double check though, it's ok to remove the `if/else` for both the `multiple=True` and the `multiple=False` versions of these arguments?

Reply:
Yep! For the other flags their input is immediately handled by the callback function which goes on to return a list, and it's this list that get's passed to `_get_values_as_tuple`, which splits them safely